src/ConsoleMain/Main.cpp

```diff
@@ -119,6 +125,15 @@ int main(int argc, char** argv)
     analysisOptions.TimelineIgnoreFunctionsUnder = std::chrono::milliseconds(timelineIgnoreFunctionsUnderMs);
     analysisOptions.TimelineIgnoreTemplatesUnder = std::chrono::milliseconds(timelineIgnoreTemplatesUnderMs);
 
+    if (timingDisplay == "ms")
+    {
+        analysisOptions.TimeDisplay = BuildAnalyzer::TimeDisplayEnum::Mili;
+    }
+    else if (timingDisplay == "s")
+    {
+        analysisOptions.TimeDisplay = BuildAnalyzer::TimeDisplayEnum::Sec;
+    }
```

Comment:
Can we show a message when we get other option saying we'll use the default representation (and which one)?

Reply:
Done.

Reply:
Can you please add the case where user passes `ns` as the time unit?
It's currently displaying the _Unrecognized time display value_ although `ns` is a valid one.
Thank you!